app/views/LicensorViewComponent.vue

```diff
@@ -283,7 +283,7 @@ module.exports = Vue.extend({
   created: ->
     return unless me.isAdmin() or me.isLicensor()
     api.apiClients.getAll().then (clients) =>
-      @ownedClients = clients.filter((c) -> c.creator is me.id)
+      @ownedClients = clients
```

Comment:
@shubhi1092 @chethana-sastry it seems like we should rename `getAll()` to `getOwned()` next time we're updating this code.  As written, this change is a little misleading and suggests a non-admin licensor may be getting access to all clients even though they are not.